src/components/dashboard/settings/client-settings-form.tsx

```
@@ -42,67 +42,67 @@ export default function ClientSettingsForm(props: {
+        <form action={formAction} className="container mx-auto flex flex-col gap-5">
+            <div className="flex items-center justify-between">
+                <h1 className="text-2xl font-bold">Artist Settings</h1>
+                <SubmitButton fieldChanged={fieldChanged} />
+            </div>
+            <Separator />
+            <div>
+                <div className="form-control">
+                    <Label className="label">Header Photo</Label>
+                    <UpdateHeaderDropzone />
+                </div>
+                <div className="divider"></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)